oauth2/oauth2-client/src/test/java/org/springframework/security/oauth2/client/registration/ClientRegistrationTests.java

```diff
@@ -42,6 +44,13 @@
 	private static final String TOKEN_URI = "https://provider.com/oauth2/token";
 	private static final String JWK_SET_URI = "https://provider.com/oauth2/keys";
 	private static final String CLIENT_NAME = "Client 1";
+	private static final Map<String, Object> PROVIDER_CONFIGURATION_METADATA = new LinkedHashMap<>();
+
+	static {
```

Comment:
Please extract initialization into a method.

Reply:
This isn't truly final because Map is mutable. Please change the Map to be immutable or (preferably) remove the static modifier.